### ISSUE TITLE
Snip wont start

### ISSUE THREAD

**Issue by leonoctem** [dlrudie/Snip#445]
I've been using snip for a couple of years no with no issue (except for being sad when it stopped supporting winamp), but today, when I try and start snip, the icon will pop up for a moment, then disappear. It's not running at all when I look in task manager either. I've tried deleting the snip folder and re-downloading, but the same thing happens.

**Comment by midnight-studios** [comment]
I can confirm the same issue. Snip (v7.0.5) is unusable.

**Comment by leonoctem** [comment]
installed the previous version (v7.0.4), able to run as administrator. perhaps adding a notice to 'run as administrator' when loading without doing so, instead of the program just closing without saying anything, wouldn't go amiss.

**Comment by dlrudie** [comment]
There was no public release of a v7.0.4. It went from v7.0.2 to v7.0.5.

Snip used to ask for elevated permissions. This is a bug.

However, thank you for reporting this. Snip has went through various stages of requiring administrator and not requiring administrator as Spotify has changed their API over the years. What had happened is with some of the changes to Spotify's API there were a lot of internal changes in Snip that made it no longer require administrator so I reverted the manifest change. Later on down the line Spotify changed their API and desktop client some more which required reintroduction of some code that required administrator access. I failed to re-add the requirement to the manifest file. v7.0.5 will work fine if you run it as administrator. I have implemented the change so future versions of Snip will again ask for elevated permissions.
